test/cases/parsing/evaluate/index.js

```diff
@@ -1,17 +1,28 @@
-it("should evaluate null", function() {
-	var y = null ? require("fail") : require("./a");
-	if(null)
-		require("fail");
+it("should evaluate null", function () {
+	const y = null ? require("fail") : require("./a");
+	if (null) require("fail");
 });
 
-it("should evaluate logical expression", function() {
-	var value1 = "hello" || require("fail");
-	var value2 = typeof require === "function" || require("fail");
-	var value3 = "" && require("fail");
-	var value4 = typeof require !== "function" && require("fail");
-	var value5 = "hello" && (() => "value5")();
-	var value6 = "" || (() => "value6")();
-	var value7 = (function () { return'value7'===typeof 'value7'&&'value7'})();
+it("should evaluate logical expression", function () {
+	const value1 = "hello" || require("fail");
+	const value2 = typeof require === "function" || require("fail");
+	const value3 = "" && require("fail");
+	const value4 = typeof require !== "function" && require("fail");
+	const value5 = "hello" && (() => "value5")();
+	const value6 = "" || (() => "value6")();
+	const value7 = (function () {
+		return "value7" === typeof "value7" && "value7";
+	})();
```

Comment:
```suggestion
	const value7 = (function () { return'value7'===typeof 'value7'&&'value7'})();
```

the spacing is important here

Reply:
I will take a look. a bit busy now

Reply:
pushed latest feel free to checkout if needed